From: Elvis Angelaccio <elvis.angelaccio@kde.org>
Date: Fri, 6 Jan 2017 15:35:46 +0100
Subject: Stop running executables when opening urls

This is a security risk because it's not clear when an entry in an
archive is an executable.

BUG: 374572
FIXED-IN: 16.12.1
---
 part/part.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/part/part.cpp b/part/part.cpp
index 0bb2bcb9..f71fc77b 100644
--- a/part/part.cpp
+++ b/part/part.cpp
@@ -945,7 +945,7 @@ void Part::slotOpenExtractedEntry(KJob *job)
         } else {
             KRun::runUrl(QUrl::fromUserInput(fullName, QString(), QUrl::AssumeLocalFile),
                          QMimeDatabase().mimeTypeForFile(fullName).name(),
-                         widget());
+                         widget(), false, false);
         }
     } else if (job->error() != KJob::KilledJobError) {
         KMessageBox::error(widget(), job->errorString());
